common/views/themes/utility-classes.ts

```diff
@@ -57,42 +57,10 @@ export const utilityClasses = css<GlobalStyleProps>`
     )}
   }
 
-  .is-hidden-l {
```

Comment:
I feel like the `is-hidden-` classes are a helpful visual reminder in the returned component, so I'd be tempted to keep them. 
I know these ones aren't used though. Maybe it could be a helper function, like `font()` 🤔

Reply:
Hmm, I’m not sure. I’ve backed out this change for now, and we can discuss it separately.